packages/perseus/src/__tests__/__snapshots__/renderer_test.jsx.snap

```diff
@@ -457,7 +457,7 @@ exports[`renderer snapshots incorrect answer: incorrect answer 1`] = `
                 class="default_xu2jcg-o_O-menuWrapper_wvrnr4"
               >
                 <button
-                  aria-expanded="true"
+                  aria-expanded="false"
```

Comment:
Why did this change?

Reply:
This is the correct value. The snapshot in the original PR from webapp had this wrong and I didn't notice it (the dropdown is in a closed state)